src/core/lib/security/credentials/credentials.h

```diff
@@ -100,8 +100,10 @@ void grpc_override_well_known_credentials_path_getter(
 struct grpc_channel_credentials
     : grpc_core::RefCounted<grpc_channel_credentials> {
  public:
-  // \a type is used to downcast grpc_channel_credentials objects.
-  // Implementations should make sure to use unique types.
+  // The pointer value \a type is used to uniquely identify a creds
+  // implementation for down-casting purposes. Every creds implementation should
+  // use a unique string instance for every creds instance of that creds
```

Comment:
The way this is currently phrased, it actually says the opposite of what we want here.  Saying "a unique string instance for every creds instance" means that each instance will use its own separate string.

I think what we want to say here is something like "Every creds implementation should use a unique string instance, which should be returned by all instances of that creds implementation."